### DISCUSSION TITLE
History mode configuration for Vue Router?

### DISCUSSION THREAD

**Discussion by evilkermit** [Kitware/trame#547] [Q&A] [answered]
Is there a way in Trame to configure the history mode for the Vue Router? https://router.vuejs.org/guide/essentials/history-mode

It looks like it defaults to hash mode, which is fine as a default, but I typically prefer to run things in HTML5 mode.

**Comment by jourdain** [comment, marked as answer]
Technically it is possible but we haven't enabled it due to the need of a special server setup which don't always align well with trame deployment/usage.
